runner-macros/src/entry.rs

```diff
@@ -0,0 +1,152 @@
+use proc_macro::TokenStream;
+use proc_macro2::Span;
+use quote::{quote, quote_spanned, ToTokens};
+
+enum Flavor {
```

Comment:
Why not network?

Reply:
The `RuntimeFlavor` currently doesn't have much info besides port for sandbox, but I think eventually would have a bit more info related to the context of the runtime like if its multi-threaded or not. So not necessarily just containing network info which is why I think flavor works better since its more general

Reply:
But each one is an endpoint or network.  We can always refactor later.

Reply:
yea definitely. It's an internal struct anyways so we can change it to whatever